src/hooks/useContentProtection.ts

```diff
@@ -24,21 +24,20 @@ const useContentProtection = <T>(
   }));
   const host = signingConfig?.host;
   const drmPolicyId = contentProtection?.drm?.defaultPolicyId ?? signingConfig?.drmPolicyId;
-  const drmEnabled = !!drmPolicyId;
-  const signingEnabled = !!host || !!urlSigning;
+  const signingEnabled = !!urlSigning;
 
   const { data: token, isLoading } = useQuery(
     ['token', type, id, params],
     () => {
-      // if provider is JWP
-      if (jwp && configId && !!id && signingEnabled) {
-        return getJWPMediaToken(configId, id);
-      }
       // if provider is not JWP
-      if (!!id && !!host && drmEnabled && signingEnabled) {
+      if (!!id && !!host && !!drmPolicyId) {
```

Comment:
I don't think drmPolicyID is required for the getMediaToken call to work. If it's blank, getMediaToken will do basic url signing, which we still want to support.

Reply:
aha, didn't know that basic signing (without drm) was present there  as well.